### PR TITLE
Modificacion carlos

### PR DESCRIPTION
**Pull request by Mantenal**
add code lines

### CODE REVIEW

**Review by Bryancdi — CHANGES_REQUESTED** [review]
Please add again ".PP" at the line 31, thank you.

File: sl.c [hunk 1]
```diff
@@ -73,6 +75,7 @@ void option(char *str)
             case 'F': FLY      = 1; break;
             case 'l': LOGO     = 1; break;
             case 'c': C51      = 1; break;
+            case 'b': C53      = 1; break;
```
Comment (ChrisR1999) [inline sl.c]: The value here should be 2

**Review by ChrisR1999 — CHANGES_REQUESTED** [review]
please check the comments in the file sl.c

**Review by Bryancdi — APPROVED** [review]
Good job.

**Review by ChrisR1999 — APPROVED** [review]
Fix approved, good work.

**Pull request merged**